doc/releases/mimic.rst

```diff
@@ -1,3 +1,162 @@
+v13.2.5 Mimic
+=============
+
+This is the fifth bugfix release of the Mimic v13.2.x long term
+stable release series. We recommend all mimic users upgrade.
+
+Notable Changes
+---------------
+
+* This release fixes the pg log hard limit bug that was introduced in
+  13.2.2, https://tracker.ceph.com/issues/36686. A flag called
+  `pglog_hardlimit` has been introduced, which is off by default. Enabling
+  this flag will limit the length of the pg log.  In order to enable
```

Comment:
please reduce number of spaces after the `.` from 2 to 1 (for consistency with the rest of the paragraph and with the documentation style guide)